hugegraph-core/src/main/java/org/apache/hugegraph/type/define/Cardinality.java

```diff
@@ -20,6 +20,7 @@
 import org.apache.tinkerpop.gremlin.structure.VertexProperty;
 
 /**
+ * TODO: lack license header & LICENSE reference
```

Comment:
https://github.com/JanusGraph/janusgraph/blob/master/janusgraph-core/src/main/java/org/janusgraph/core/Cardinality.java
https://github.com/JanusGraph/janusgraph/blob/master/janusgraph-core/src/main/java/org/janusgraph/core/Namifiable.java

ditto, should I Change the license to janusgraph license?should I modify the license?